plugins/hydra_ax_sweeper/tests/config/quadratic.yaml

```diff
@@ -0,0 +1,31 @@
+defaults:
```

Comment:
if this is exactly the same as plugins/hydra_ax_sweeper/tests/apps/quadratic.yaml can you reuse?

Reply:
Updated